templates/docs/examples/patterns/accordion/default.html

```diff
@@ -7,7 +7,9 @@
 <aside class="p-accordion">
   <ul class="p-accordion__list">
     <li class="p-accordion__group">
-      <button type="button" class="p-accordion__tab" id="tab1" aria-controls="tab1-section" aria-expanded="true">Advanced topics</button>
+      <div role="heading" aria-level="2" class="p-accordion__heading">
```

Comment:
Tiny thing really, but I guess many people may just copy this blindly without adjusting the `aria-level` and I'm not sure if proposing level 2 is a good default. I think within a page copy h3 or h4 is most likely to be something you expect in accordion.

Would be good for our example be such more common use case.